src/modules/pos/validator/utils/checkCommissionValidity.js

```diff
@@ -0,0 +1,12 @@
+import { convertCommissionToNumber } from './getValidatorCommission';
+
+export const checkCommissionValidity = (newCommission, oldCommission) => {
+  const newCommissionParam = convertCommissionToNumber(newCommission);
+  const oldCommissionParam = convertCommissionToNumber(oldCommission);
+
+  if (newCommissionParam < oldCommissionParam) {
+    return true;
+  }
+  const commissionIncrease = newCommissionParam - oldCommissionParam;
+  return commissionIncrease <= 500;
```

Comment:
I guess `500` should be assigned a constant so its readable to anybody seeing the snippet of code what 500 means

Reply:
@eniolam1000752 Resolved